docs/pages/versions/unversioned/sdk/multiple-app-variants.mdx

```diff
@@ -0,0 +1,214 @@
+---
```

Comment:
Please remove this file from this directory. The file you are editing is part of the EAS tutorial.